rfcs/0019-maintainers.md

```diff
@@ -0,0 +1,95 @@
+---
+feature: maintainers-file
+start-date: 2017-10-28
+author: Maarten Hoogendoorn (@moretea)
+co-authors: (find a buddy later to help our with the RFC)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Currently, nixpkgs does not have explicit maintainers for anything besides
+the packages themselves. Introducing a maintainers file enables us to mark
+maintainers for the remaining parts.
+
+# Motivation
+[motivation]: #motivation
+
+<!--  Why are we doing this? -->
+There is no explicit (machine-readable) description of who maintains what.
```

Comment:
`CODEOWNERS` file describes who owns portions.
The `maintainers` fields describe who maintain certain expressions.